src/checks/checkNodeVersion.js

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS 
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+async function checkNodeVersion(metadataJSON) {
+    if ('_nodeVersion' in metadataJSON) {
+        var nodeVersion = metadataJSON._nodeVersion
+        const regex14 = new RegExp('^14(.[0-9]d*)*')
+        const regex16 = new RegExp('^16(.[0-9]d*)*')
```

Comment:
This is fine for now, but what happens when the supported versions change? We'll surely need to actively maintain these lines.

Reply:
@roribio I updated checkNodeVersion to have a const array of supported versions, and the function will loop through the array and run a regex on each.